geode-wan/src/test/java/org/apache/geode/internal/cache/wan/WANTestBase.java

```diff
@@ -3131,10 +3210,10 @@ public static Integer getQueueContentSize(final String senderId, boolean include
         return ((ParallelGatewaySenderQueue) regionQueue).localSize(includeSecondary);
       } else {
         if (includeSecondary) {
-          fail("Not Implemented yet");
+          fail("Not implemented yet");
```

Comment:
This test should be failing if this line is present... the below lines would never be called

Reply:
The whole "else" will never be executed. I have confirmed with Barry. 

I will change it to be:
} else {
  fail("Not Implemented yet");
}